src/workerd/api/queue.c++

```diff
@@ -174,7 +174,8 @@ jsg::JsValue deserialize(jsg::Lock& js, rpc::QueueMessage::Reader message) {
 
 kj::Promise<void> WorkerQueue::send(jsg::Lock& js,
                                     jsg::JsValue body,
-                                    jsg::Optional<SendOptions> options) {
+                                    jsg::Optional<SendOptions> options,
+                                    CompatibilityFlags::Reader flags) {
```

Comment:
Note that this is no longer really necessary. You can grab the feature flags also using the `FeatureFlags` type (see `/workerd/io/features.h`)

Reply:
thanks for the tip! updated to use this approach